Sources/Widgets/Widgets3D/ShapeWidget/behavior.js

```diff
@@ -371,29 +371,24 @@ export default function widgetBehavior(publicAPI, model) {
     ) {
       return macro.VOID;
     }
-
+    const normal = model.camera.getDirectionOfProjection();
     if (!model.point2) {
-      // Update orientation to match the camera's plane
-      // if the corners are not yet placed
-      const normal = model.camera.getDirectionOfProjection();
```

Comment:
@ocrossi at first glance I don't agree with this (hence why the comment was here). After the 1st point is placed, the 2nd anchor point should be placed to keep the shape parallel to the camera. However, if the 2nd point is already placed, the shape widget plane shouldn't change based on the camera (ex: if you placed the widget on a slice and rotate the camera, that widget shouldn't be able to start facing another direction than the slice).

Can you help me understand what you're trying to do and what scenario would be fixed by that change?

Reply:
the problem encountered here was to place the widget on the good slice. When the user would change slices in the UI the widgetBehavior would not set correct handle position and kept the slice value of when the camera was placed (while initialization of the scene or when you would change slice mode) 
This fix was meant to update the manipulator position from the example when the user changes slices. It fixed this problem. 
I did not consider the problem of changing view while placing widget.

Reply:
You are right the orientation should keep the state of the plane facing the camera when the user places point1

Reply:
so we're reverting this?

Reply:
So we're reverting this?

Reply:
No, my mistake for leaving this here